src/components/NavigationMenu.tsx

```diff
@@ -2,13 +2,22 @@ import React from 'react';
 
 import '../styles/components/NavigationMenu.css';
 
-export default function NavigationMenu(){
+import SiteLogo from '../components/SiteLogo';
+
+interface NavigationProps {
+    useLogo: boolean
+}
+
+export default function NavigationMenu(props: NavigationProps) {
     return (
-        <div className="menu">
-            <a href="">Home</a>
-            <a href="">About us</a>
-            <a href="">Our tries</a>
-            <a href="">Contribute</a>
+        <div id="navigation-menu">
+            {props.useLogo && (<SiteLogo className="site_menu" width={162} height={70} />)}
```

Comment:
Please, enlighten me. What does this lone do?

Reply:
This is basically a validation, that represents the following lines:

`if(props.useLogo){
    <SiteLogo className="site_menu" width={162} height={70} />
}`

So the && makes this comparison shorter. It's called conditional render, and you can read more about it [here](https://pt-br.reactjs.org/docs/conditional-rendering.html)

Reply:
It's nice to lean about, because you could do very interesting things with it, like:

`{
    (props.useLogo ? 
        (<SiteLogo className="site_menu" width={162} height={70} />) 
    : 
        (<SiteLogo className="site_menu" width={70} height={70} />))
}`

Reply:
I see... We set the action of displaying the SiteLogo as one side of a boolean comparison. If props.useLogo is false, then the comparison is short circuited into false and the rendering is never executed. Thanks @claytonrv, that's new frontend stuff to me.